src/Relude/Lifted/Concurrent.hs

```diff
@@ -54,7 +54,13 @@ import qualified Control.Monad.STM as STM (atomically)
 -- Lifted Control.Concurrent.MVar
 ----------------------------------------------------------------------------
 
--- | Lifted to 'MonadIO' version of 'CCM.newEmptyMVar'.
+{- | Lifted to 'MonadIO' version of 'CCM.newEmptyMVar'.
+
+TODO: newEmptyMVar
+not sure if should create a top level doc or give examples for each.
+Just the original docs are descriptive but don't give code examples.
```

Comment:
I guess top-level docs for these functions are okay. I'm not sure how to create examples for these functions... And probably it's better to start with `IORef`. And then the same can be implemented for `MVar`

Reply:
there are links to 'MVar' and 'STM' in top description not sure if that suffices.